c2corg_api/views/__init__.py

```diff
@@ -177,12 +180,21 @@ def set_best_locale(documents, preferred_lang, expunge=True):
 
 
 def get_best_locale(available_locales, preferred_lang):
-    if preferred_lang in available_locales:
-        best_locale = available_locales[preferred_lang]
-    else:
+
+    print(' LOOOOOOOOOOOOOOOOOOOOOOOG ', preferred_lang, available_locales)
+
+    if type(preferred_lang) is list:
```

Comment:
I think it would be good to allow users to select more languages in the feed. I think that you too.
![image](https://cloud.githubusercontent.com/assets/6165660/23402989/3e9fd210-fdae-11e6-849a-118d3639c9c9.png)
It would be good later to allow users to change a priority of the best locale which shows in the feed (right now it is handled by the `get_best_locale` [function](https://github.com/c2corg/v6_api/blob/179b1d45baf224c1fd80a937f8c83714b689d0b4/c2corg_api/views/__init__.py#L179) which returns the best language based on the `lang_priority` [attribute](https://github.com/c2corg/v6_common/blob/1e3fe0991ecc8acbb0d9234f96eb71413089aa7f/c2corg_common/attributes.py#L3)).